R/dplyrVerbs.R

```diff
@@ -75,9 +95,18 @@ left_join.qseaSet <- function(x, y, by = NULL, copy = FALSE, suffix = c(".x",".y
 #' @importFrom dplyr select
 #' @param .data A qseaSet to select columns of the sampleTable from
 #' @param ... Other arguments to pass to dplyr::select
-#' @return A qseaSet object with the sampleTable changed by a call to dplyr::select
+#' @return A qseaSet object with the sampleTable changed by a call to dplyr::select. 
+#'   The sample_name and group columns will always be returned.
 #' @export select.qseaSet
 #' @export
+#' @examples
```

Comment:
Hmm. I think it is unnecessary to give a warning message, think it is neater to just include them. Otherwise you'd get a warning message every time unless you explicitly add them.